### ISSUE TITLE
[BUG] Loading bar, title - everything showing.

### ISSUE THREAD

**Issue by soenkegissel** [techinessoverloaded/progress-dialog#16]
Using API 28 Simulator and Samsung S10 API 31.
Init like this:
```
ProgressDialog progressDialog;
if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
            progressDialog = new ProgressDialog(requireContext(), ProgressDialog.THEME_FOLLOW_SYSTEM);
        } else {
            progressDialog = new ProgressDialog(requireContext(), ProgressDialog.THEME_LIGHT);
        }
progressDialog.show()
```

Same for both devices:
![image](https://user-images.githubusercontent.com/13346316/171257605-468b53a8-5000-443b-bf78-f3d4eebae84a.png)

**Comment by techinessoverloaded** [comment]
Hi, 
Thanks for pointing out an issue. May I know which Version of the Library you are using ?

**Comment by soenkegissel** [comment]
1.4.4

**Comment by techinessoverloaded** [comment]
Ok, I will check out the issue and resolve it in the upcoming version.

**Comment by techinessoverloaded** [comment]
Anyway, I will check out if the issue is on my code. If it is so, I will resolve it in the next version.

**Comment by mg2000** [comment]
I have same problem.

**Comment by techinessoverloaded** [comment]
> I have same problem.

Hi, I will fix that problem in the upcoming version. Please use version 1.3.1 until then.

**Comment by techinessoverloaded** [comment]
@soenkegissel @mg2000 I have fixed this bug in version 1.5.1.
